ui/app/pages/permissions-connect/choose-account/index.scss

```diff
@@ -1,34 +1,87 @@
 .permissions-connect-choose-account {
   display: flex;
   flex-direction: column;
-  margin-top: 40px;
-  width: 100%;
   align-items: center;
+  margin-top: 40px;
+  width: 428px;
+  margin-left: auto;
+  margin-right: auto;
 
   &__title {
     @extend %header--18;
   }
 
-  &__text {
+  &__text, &__text--blue, &__text--grey {
     @extend %content-text;
     line-height: 25px;
   }
 
+  &__text-blue {
+    color: $curious-blue;
+    cursor: pointer;
+  }
+
+  &__text-grey {
+    color: $Grey-500;
+  }
+
   &__accounts-list {
-    width: 393px;
+    width: 100%;
     border: 1px solid #D0D5DA;
     box-sizing: border-box;
     border-radius: 8px;
-    margin-top: 36px;
+    margin-top: 8px;
+    max-height: 338px;
+    overflow-y: auto;
 
     @media screen and (max-width: 575px) {
       width: 100%;
     }
   }
 
+  &__accounts-list-header {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 36px;
+    width: 100%;
+    padding-right: 2px;
+  }
+
+  &__select-all {
+    display: flex;
+    margin-left: 16px;
+    align-items: center;
+  }
+
   &__account-info-wrapper {
     display: flex;
     justify-content: flex-start;
+    align-items: center;
+  }
+
+  &__list-check-box {
+    margin-right: 24px;
+  }
+
+  &__header-check-box {
+    margin-right: 16px;
+  }
+
+  &__select-all {
```

Comment:
Nit: Why are there two sections for this class? It's defined on line 50 above as well.

Reply:
fixed in 47c70ea56